pointer-events-2015.html

```diff
@@ -136,17 +136,52 @@ <h1 id="title">Pointer Events Working Group Charter</h1>
 
-        <p>W3C has two competing standards for defining touch interfaces: the <a href="http://www.w3.org/TR/touch-events/">Touch Events specification</a> and the <a href="http://www.w3.org/TR/pointerevents/">Pointer Events 1.0 specification</a>. Each has browser implementation, dependent script libraries, and developer uptake. This Working Group will reconcile the relationship and best features between these technologies.</p>
+        <p>Pointer Events provide support for handling mouse, touch, and pen input for web sites and web applications through DOM Events.For example, a content creator using Web Pointer Events would have only use a single model, rather than separate code paths for mouse events, touch events, and pen-tablet events, making authoring content much more efficient and inclusive.</p>
 
-        <p>Among the features under discussion are the touch-action CSS property, behavior in sending touch events when scrolling starts, and the relationship and mappings between mouse events, touch events, and pointer events.</p>
+        <p>This Working Group seeks to enhance the features delivered in the Pointer Events Level 1 Recommendation by exploring changes, such as:</p>
+          <ul>
+            <li>Add direction-specific values for finer=grained control of panning touch behaviors</li>
+            <li>Reducing hit-testing via implicit capture</li>
+            <li>Additional clarifications of API behavior and performance optimizations</li>
+          </ul>
 
         <div id="section-out-of-scope">
           <h3 id="out-of-scope">Out of Scope</h3>
             <p>The following features are out of scope, and will not be addressed by this working group.</p>
-            
+
             <ul class="out-of-scope">
+              <li>Gestures. Examples of out-of-scope gesture functionality and APIs include, but are not limited to, the following:
+                <ul>
+                  <li>Comparisons between pointers to determine an action (e.g., panning for scrollable regions, pinch for zooming, press-and-hold for a mouse right-click).</li>
+                  <li>Comparisons between time stamps of pointers to determine an action.</li>
+                  <li>Comparisons between combinations of pointers and/or their time stamps to determine an action.</li>
+                  <li>Determining an action based on comparison to a threshold (e.g., scroll speed based on a pressure threshold, panning based on distance threshold, press-and-hold based on a timing threshold).</li>
+                  <li>APIs or functionality processing data that is indicative of a confidence level that a pointer is associated with a gesture.</li>
+                </ul>
+              </li>
+
+              <li>Higher level APIs used to convey user intent.
+                <ul>
+                  <li>High-level representational events, which are in the scope of the Web Events and IndieUI working groups.</li>
```

Comment:
I think they're ok to keep because those groups' charters exist which help serve as an example of what's out of scope.